gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -111,6 +111,7 @@ def construct_argparser(self) -> argparse.ArgumentParser:
         parser.add_argument(
             "--share_intrinsics", action="store_true", help="Shares the intrinsics between all the cameras"
         )
+        parser.add_argument("--mvs_off", action="store_true", help="Turn off dense MVS reconstruction")
```

Comment:
nit: For flags it is more intuitive to have default false with the flag option to turn on. 
so having a flag like "--run_mvs, default = False" would let us do --run_mvs when we want to run it.  now we have to do --mvs_off=0 for running mvs.

Reply:
I believe having it this way will run it by default, and it can be turned off by calling `--mvs_off`, correct? I can change it to no run by default if you'd like.

Reply:
if the default is true, yeah.